### PR TITLE
Added the four main Glyph categories and updated the README file

### PR DESCRIPTION
**Pull request by goksel**

### CODE REVIEW

File: Ontology/v2/sbol-vo.rdf [hunk 1]
```diff
@@ -47,25 +42,34 @@
...
-      <owl:someValuesFrom rdf:resource="http://sbols.org/v2#Interaction"/>
-    </owl:Restriction>
-  </rdfs:subClassOf>
-  <rdfs:subClassOf>
-    <owl:Restriction>
-      <owl:onProperty rdf:resource="#isGlyphOf"/>
-      <owl:someValuesFrom>
-        <owl:Restriction>
-          <owl:onProperty rdf:resource="http://sbols.org/v2#type"/>
-          <owl:someValuesFrom rdf:resource="http://identifiers.org/sbo/SBO:0000169"/>
```
Comment (jakebeal) [inline Ontology/v2/sbol-vo.rdf:64]: This change seems to have removed all of the SBO restrictions for interactions; we don't want to lose that.

**Review by jakebeal — CHANGES_REQUESTED** [review]
SBO restrictions appear to have been lost for interactions.

File: Ontology/v2/sbol-vo.rdf (hunk 1 above)
Comment (jakebeal) [reply Ontology/v2/sbol-vo.rdf:64]: The same is the case for all of the other interactions

**Comment by jakebeal** [comment]
Note: addressing #96

**Comment by goksel** [comment]
@jakebeal Please check merge the pull request https://github.com/SynBioDex/SBOL-visual/pull/133.

**Comment by jakebeal** [comment]
I've added a comment on #133, and am waiting for your response there.

Can this one be closed as obsolete?

**Comment by jakebeal** [comment]
Superseded by #133. @goksel , can you delete the branch if it is no longer needed?

**Pull request closed without merging**